client/src/App.jsx

```diff
@@ -45,6 +46,13 @@ class App extends Component {
                 </div>
               )}
               />
+              <Route exact path="/orders" render = {() => (
+                <div>
+                <Sidebar />
```

Comment:
@denaHS If Sidebar and Navbar are designed to appear on many of your routes, you could move them up a few levels above the `<Switch>` or you could create a HOC (higher-order component) to "wrap" your different page components that should have them, so that you don't have to re-write `<Sidebar />` and `<Navbar />` inside App.js

Reply:
thanks, we will do it in a separate issue 
@hshahwan @Jamalat-shamallakh @Anies12 @naremanhilles @denaHS